### ISSUE TITLE
ENet prediction become worse after 100 epoch

### ISSUE THREAD

**Issue by guanqp** [xiaoyufenfei/Efficient-Segmentation-Networks#17]
I use this project to train ENet on cityscapes dataset(640*480). The loss decase normally, but predict result become worse when trainning for more than 50 epochs...
![image](https://user-images.githubusercontent.com/5735046/109457850-a103dc80-7a96-11eb-876d-752a5fc42895.png)
from left to right, 10epoch, 50epoch, 80epoch.
and the training loss vs epochs looks fine:
![image](https://user-images.githubusercontent.com/5735046/109457958-d90b1f80-7a96-11eb-9842-f9db7098a38e.png)

Can anyone give me some advice? Thanks!

**Comment by guanqp** [comment]
Looks like the train process does not evaluate the validation loss and mIoU, so the model over fited.
